### ISSUE TITLE
Insert references for sources of contamination

### ISSUE THREAD

**Issue by tomschenkjr** [Chicago/predicting-e-coli-concentrations#39]
> Line 77- Insert references for sources of contamination- there are CSOs, septic systems, agricultural run-off etc are all contaminants in the great lakes… See papers from Rose et al. et Canadian researchers

**Comment by tomschenkjr** [comment]
For my own notes, here are papers that cite the non-point pollution for Chicago:
  * From waves: [Ge, Z., Whitman, R. L., Nevers, M. B., & Phanikumar, M. S. (2012). Wave-Induced Mass Transport Affects Daily Escherichia coli Fluctuations in Nearshore Water. Environmental Science & Technology, 46(4), 2204–2211.](https://www.zotero.org/groups/187068/city_of_chicago_data_science_team/items/collectionKey/RJJ5U3BC/itemKey/CMRRVTGD/q/Ge) 
  * Great Lakes watershed in [Whitman, R. L.; Nevers, M. B.; Byappanahalli, M. N. Watershedwide distribution of Escherichia coli along southern Lake Michigan: An integrated approach. Appl. Environ. Microbiol. 2006, 72, 7301–7310.](http://aem.asm.org/content/72/11/7301.full)
  * Sand/water iteraction in [Foreshore sand as a source of
Escherichia coli in nearshore water of a Lake Michigan beach.
Appl. Environ. Microbiol. 2003, 69, 5555–5562.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC194947/)

**Comment by tomschenkjr** [comment]
I've discovered an issue with the `issue50` branch. Zotero apparently made a change so when I exported one of the citations to BibTeX, the BibTeX file started to use an "&" in the name-entry. This is an illegal character in Pandoc, which causes the build the fail.

I've simply corrected the entry in Zotero, but just need to re-export to the `bibliography/zotero-references.bib` file to resolve it on the branch.
